src/components/Sidebar/RightSidebar.tsx

```
@@ -0,0 +1,150 @@
+    headerElements.forEach((header) => {
+      observer.current?.observe(header);
+    });
+
+    return () => {
+      observer.current?.disconnect();
+    };
+  }, []);
+
+  const highlightPosition = useMemo(() => {
+    const sidebarElement =
+      typeof document !== `undefined` ? document.getElementById(`sidebar-${activeHeader?.id}`) : null;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)